api/src/main/java/org/openmrs/PersonName.java

```diff
@@ -530,42 +531,12 @@ public String toString() {
 	 * @should return negative if other familynamePrefix is greater
 	 * @should return negative if other familyNameSuffix is greater
 	 * @should return negative if other dateCreated is greater
+	 * @Depracated since 1.12. Use DefaultCompartor instead.
```

Comment:
Could we please add in addition to @Deprecated the following:
Note: this comparator imposes orderings that are inconsistent with equals.